damus/Models/Contacts.swift

```diff
@@ -58,6 +60,10 @@ class Contacts {
     
     func remove_friend(_ pubkey: String) {
         friends.remove(pubkey)
+
+        pubkey_to_our_friends.forEach {
```

Comment:
This can be inefficient if the user has a large following list, but I didn't want to create yet another meta data structure for the sake of lookups specifically for removal. We can always optimize later.